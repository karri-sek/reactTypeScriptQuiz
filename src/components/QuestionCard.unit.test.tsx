import React from "react";
import { shallow } from 'enzyme';
import QuestionCard from './QuestionCard';

describe('QuestionCard', () => {
  it('Renders Question ', () => {
    const questionCard = shallow(<QuestionCard
      questionNr={1}
      totalQuestions={10}
      question={'sample Question'}
      answers={['here is the answer']}
      userAnswer={undefined}
      callback={() => { }}
    />);
    expect(questionCard).toMatchSnapshot();
  });
});