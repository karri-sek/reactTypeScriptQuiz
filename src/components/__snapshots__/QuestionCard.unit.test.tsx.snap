// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionCard Renders Question  1`] = `ShallowWrapper {}`;
